01_prepare_host.sh

```diff
@@ -27,9 +27,16 @@ if [[ $OS == ubuntu ]]; then
     sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.10 1
     # (workaround) disable tdp_mmu to avoid
     # kernel crashes with  NULL pointer dereference
-    sudo modprobe -r -a kvm_intel kvm
-    sudo modprobe kvm tdp_mmu=0
-    sudo modprobe -a kvm kvm_intel
+    # note(elfosardo): run this only if we have kvm support
+    if grep -q vmx /proc/cpuinfo; then
+      sudo modprobe -r -a kvm_intel kvm
+      sudo modprobe kvm tdp_mmu=0
```

Comment:
mmmm not sure this can be done without complicating the code too much, we would save one line but probably add more, we need to disable the kvm module first, and only if we detect virtualization support (either intel or amd)